bundles/org.openhab.core/src/main/java/org/openhab/core/items/events/ItemStateChangedEvent.java

```diff
@@ -62,6 +60,7 @@ public String getType() {
      *
      * @return the item name
      */
+    @Override
     public String getItemName() {
```

Comment:
This can be removed as it the same like defined in the abstract `ItemEvent` class.